### ISSUE TITLE
Stremio app not playing any videos on Samsung TV - "Stream failed to load"

### ISSUE THREAD

**Issue by sebastn** [Stremio/stremio-bugs#780]
**Describe the bug**
Stremio app on Samsung fails play any content

**To Reproduce**
1. Go to Samsung app store and install Stremio
2. Click on Stremio app and link your account (in my case - Stremio + RD + MF + Torrentio)
3. Scroll down to Discover/Library and play any video.
4. Player waiting to load for 1-2 minutes and finally shows the error "Stream failed to load"

**Expected behavior**
Video to buffer and play after 2-5 seconds 

**TV (please complete the following information):**
 - Device: Samsung GU55BU8079
 - OS: [e.g. Tizen 6.5 (Samsung)]
 - Version - Theater v1.3.0

**Additional context**
Amazon firetvstick 4k connected to the TV streams videos without much issues. 
App version - 1.6.9 ARM Apk

**Comment by jaruba** [comment]
force restart ur TV by keeping the power button pressed until you see the Samsung logo

**Comment by jaruba** [comment]
Also, it should be mentioned that you should not compare the Android TV app with the Smart TV apps.

The Android TV app uses Media3, which supports all audio / video decoders, while Samsung and LG only support what the TV official video player from these brands support. There is no alternative of using any other video player on these devices.

**Comment by sebastn** [comment]
Restarting the TV didn't help. May be it has to do something with the Samsung app store version of the app. Anyway thanks for the response. :)

**Comment by jaruba** [comment]
i don't think that the version has anything to do with it, the TV will only play what the Samsung TV model supports, but decoding support can be different from one model year to another

**Comment by htoomyintnaung** [comment]
I have the exact same problem with my QN95B. Force restart or even unplug and plug doesn't help. I cannot play any of the videos.

**Comment by tomersiv** [comment]
Also encounter the same problem with Stremio+Torrentio and Samsung S90C TV.

**Comment by htoomyintnaung** [comment]
@jaruba What me and @sebastn trying to say is, we cannot play ANY of the videos on the Samsung TV Stremio app. Maybe  said "Amazon firetvstick 4k connected to the TV streams videos without much issues" to clarify that this is not a Torrentio or RD configuration problem. If you have any clue, please let us know how.

**Comment by sideskroll** [comment]
![PXL_20240813_085221539](https://github.com/user-attachments/assets/bffa3cbb-3c8f-4dae-b492-8fce44657bcb)
Same here.

**Comment by jaruba** [comment]
maybe some DNS blocking? i believe u can change the dns provider in the network settings, maybe try changing to cloudflare or google dns?

**Comment by xalaros** [comment]
Same problem to me with model 2019.
Changed DNS to cloudflare, restart tv but didn't solve the problem.

**Comment by jaruba** [comment]
do 1080p streams work?

**Comment by sideskroll** [comment]
> maybe some DNS blocking? i believe u can change the dns provider in the network settings, maybe try changing to cloudflare or google dns?

At least in MY case I have cloudflare as default at router level. I also, tried with googles and my own isp just in case. It has nothing to do with that.

**Comment by Knufle** [comment]
Same problem here

**Comment by jaruba** [comment]
nobody is answering my question, do 1080p streams work?

**Comment by sideskroll** [comment]
> nobody is answering my question, do 1080p streams work?

No. Nothing. 480, 720, 1080. It just doesn't load.
And it isn't codec related either.
Not even Avi containers work.

**Comment by Nikanaphros** [comment]
Did they fix it?

**Comment by makobot-sh** [comment]
I started having the same issue on a Samsung CU7050 yesterday, no movies work anymore even movies that worked two days ago

**Comment by alexsilvasss** [comment]
Estou com o mesmo problema aqui, reinstar o aplicativo e reiniciar a TV não resolveu, nas configurações em servidor aparece " off-line " e não tem como corrigir

**Comment by romulosimiquel** [comment]
Using the option reinstall the app when holding select on app icon worked for me.

**Comment by dr0vur** [comment]
I have started having the same issue for the past 2 weeks. Everything was working fine  before. Tried everything no luck... I don't understand why.

I reset TV
I Removed and reinstall app
I tried dns change
I tried force restart

Any other ideas of what could be the cause of this?

**Comment by felps-dev** [comment]
Same problem over here... It started few days ago, I can normally use Stremio on the notebook / PC, but on the samsung TV I face the problem described.

**Comment by htoomyintnaung** [comment]
Coming back to update my situation hoping it could help diagnose the problem. I had this problem since 6 months ago and tried all those uninstall, reinstall, reboot, change dns solutions and none of them worked. However a month ago, I switched to a new ISP at my home and somehow suddenly everything working now. It's the same TV, same configuration, same router that hasn't been working for months suddenly working after changing to new ISP. This is somehow related to ISPs blocking some ports related to torrent trackers?

**Comment by felps-dev** [comment]
> Coming back to update my situation hoping it could help diagnose the problem. I had this problem since 6 months ago and tried all those uninstall, reinstall, reboot, change dns solutions and none of them worked. However a month ago, I switched to a new ISP at my home and somehow suddenly everything working now. It's the same TV, same configuration, same router that hasn't been working for months suddenly working after changing to new ISP. This is somehow related to ISPs blocking some ports related to torrent trackers?

But how it works on my Mac and phone? This doesn't explain that... if there's some packet blocking stuff, it would not work on any device.

**Comment by dr0vur** [comment]
I suspect that the 100 Mbps network card of the TV might be the bottleneck, especially for demanding 4K content. This is puzzling, as I've been able to stream 4K content without problems using the Stremio TV app in the past.

As a potential solution, I'm considering investing in a dedicated streaming device with a 1 Gbps Ethernet port to improve performance and ensure a more reliable 4K experience.

**Comment by AnastasiyaLoichyts** [comment]
The only thing that worked was resetting network and reconnecting to the Internet.

**Comment by LironD** [comment]
A solution has been found! The TV needs to be rebooted. Attached is a link explaining the process:
https://www.samsung.com/uk/support/tv-audio-video/how-to-cold-boot-reboot-restart-or-reset-your-samsung-smart-tv/?srsltid=AfmBOoqrzXXk_QiRGUygdcw_zHo5ExfdEyTPzNEVlzrNMLXMOcLdau6K

**Comment by YaliB** [comment]
> A solution has been found! The TV needs to be rebooted. Attached is a link explaining the process: https://www.samsung.com/uk/support/tv-audio-video/how-to-cold-boot-reboot-restart-or-reset-your-samsung-smart-tv/?srsltid=AfmBOoqrzXXk_QiRGUygdcw_zHo5ExfdEyTPzNEVlzrNMLXMOcLdau6K

I experienced the same problem noted in this thread. After trying all the suggested solutions I can **verify that this solution did work** for me after none of the other solutions did. 👍

**Comment by sideskroll** [comment]
> > A solution has been found! The TV needs to be rebooted. Attached is a link explaining the process: https://www.samsung.com/uk/support/tv-audio-video/how-to-cold-boot-reboot-restart-or-reset-your-samsung-smart-tv/?srsltid=AfmBOoqrzXXk_QiRGUygdcw_zHo5ExfdEyTPzNEVlzrNMLXMOcLdau6K
> 
> I experienced the same problem noted in this thread. After trying all the suggested solutions I can **verify that this solution did work** for me after none of the other solutions did. 👍

No it doesn't. I mean, it very well could. But the issue resolved itself. No input on my part. So I think it was something else...

**Comment by lionelresnik** [comment]
Rebooting fixes the issue momentarily, it appears short after again

**Comment by consecrated** [comment]
Same problem. Has anyone found a solution?

**Comment by consecrated** [comment]
The only solution I found was to download the browser on the Samsung TV and access it through the link: app.strem.io
It worked this way for me.

**Comment by JPTTPO** [comment]
You can see the ones that are the pirate bay

**Comment by DadyKay50** [comment]
Same problem for Samsung Tizen os but there is something else i found bizarre if someone can hep, i just bought the Tv but the moment i start using it i found it too heavy, ans specially stremio the app is so so heavy and laggy

**Comment by Jiepmcm** [comment]
Log in to your stremio account on the tizen browser and leave it running (as in press the home button). Then go to the app and select whichever movie/series you were having a problem with and it should be working 👍

**Comment by TRtomasz** [comment]
We have released an update app version, should be fixed now

**Comment by BerkLogun** [comment]
> We have released an update app version, should be fixed now

When will we see the results of the TV update? The Samsung TV App Store shows it was updated on January 13, 2025.

**Comment by magnolett** [comment]
doesn't seems to be fixed, i have a Samsung 55QN85D Tizen 8 but everytime i need to clear cache and data to unstuck the infinite loading page.

**Comment by jaruba** [comment]
@magnolett if you go to the app settings, then "About", then "Platform" does it say "Tizen 8"?

do you use torrents? when it gets "stuck" does it say that the server is offline in the Stremio app settings? does doing a hard reboot of the TV (by keeping the power button pressed for a long time until you see the samsung tv logo) fix the video loading?

**Comment by jaruba** [comment]
@BerkLogun you are confusing shell version with ui version, to see the ui version of the app, open Stremio and go to Settings > About > Version

**Comment by BerkLogun** [comment]
hey @jaruba here is the version info of the tv and stremio

platform -> Tizen 6.5
version -> Theater v1.6.4

and on the server section it says v4.20.8 and its online

i guess my tv is just old ? haha

thanks for the reply btw love the product and im screencasting from my laptop to enjoy the shows so no worries if the problem is on my end

**Comment by mariametodieva** [comment]
hi, same issue here, platform is Tizen 8, version is Theater v1.6.4. I've tried rebooting, clearing cache, uninstalling, basically everything. Stremio was working fine up until yesterday, when all of a sudden my movie just failed to load and started showing the failed message after few seconds of buffering.

**Comment by NevetsCJ** [comment]
@jaruba @TRtomasz could this be reopened please? This is a great app, but this bug is rendering it unusable on some TVs for some people. There have been posts on r/Stremio regarding this bug recently too (post update)

**Comment by jaruba** [comment]
@NevetsCJ i don't believe it's fixable, the initial issues were for torrent playback (the stremio server being killed) but ur report is for https streams

i also have a 2019 old TVs that is capable of playing more streams than some cheap 2021 TVs, the 2021 TVs has issues with a lot of 4k streams, but 1080p streams are fine, a different team mate has a more expensive 2021 TV that has no such issues

i tried to debug the issues with my TVs but they proved to be unsolvable, the samsung tv video player just errors out saying that the stream is not supported by the device

if the samsung tv video player refuses to play a https stream it is likely that the issue is with what codecs that TV supports, which cannot be fixed from our side

**Comment by NevetsCJ** [comment]
Damn, that really is a shame. I’ll get a Firestick or something for my other TV. Thank you for all the work you do on Stremio, it really is a great app

**Comment by Stroumpoul** [comment]
> hey [@jaruba](https://github.com/jaruba) here is the version info of the tv and stremio
> 
> platform -> Tizen 6.5 version -> Theater v1.6.4
> 
> and on the server section it says v4.20.8 and its online
> 
> i guess my tv is just old ? haha
> 
> thanks for the reply btw love the product and im screencasting from my laptop to enjoy the shows so no worries if the problem is on my end

Guys i think i found it just go to settings -server-online and hit reload

**Comment by sergueidob** [comment]
Still have issues on one of the TVs. 

server 4.20.8
theater 1.7.2
Platform tizen6

**Comment by godofthunder8756** [comment]
Suddenly the same thing just happened to me. Cold booting my Samsung TV and reinstalling the app did not work. It was working yesterday. No updates or anything I'm aware of. Using Stremio + Torrentino + RD

**Comment by GameDevUmut** [comment]
> Suddenly the same thing just happened to me. Cold booting my Samsung TV and reinstalling the app did not work. It was working yesterday. No updates or anything I'm aware of. Using Stremio + Torrentino + RD

The same thing happened to me as well. I have the same setup with you. 

It works on pc & android phone on the same network but it doesn't work on my tizen tv. It stopped working this week. It gets stuck at loading screen when I want to load any stream, then shows "stream failed to load" message.

**Comment by sergueidob** [comment]
> > Suddenly the same thing just happened to me. Cold booting my Samsung TV and reinstalling the app did not work. It was working yesterday. No updates or anything I'm aware of. Using Stremio + Torrentino + RD
> 
> The same thing happened to me as well. I have the same setup with you.
> 
> It works on pc & android phone on the same network but it doesn't work on my tizen tv. It stopped working this week. It gets stuck at loading screen when I want to load any stream, then shows "stream failed to load" message.

Try unpluggin the power cord and wait a few seconds, that did the trick for me. Using the remote control IS NOT THE SAME.

The TV must remain some time without power.

**Comment by GameDevUmut** [comment]
> > > Suddenly the same thing just happened to me. Cold booting my Samsung TV and reinstalling the app did not work. It was working yesterday. No updates or anything I'm aware of. Using Stremio + Torrentino + RD
> > 
> > 
> > The same thing happened to me as well. I have the same setup with you.
> > It works on pc & android phone on the same network but it doesn't work on my tizen tv. It stopped working this week. It gets stuck at loading screen when I want to load any stream, then shows "stream failed to load" message.
> 
> Try unpluggin the power cord and wait a few seconds, that did the trick for me. Using the remote control IS NOT THE SAME.
> 
> The TV must remain some time without power.

Thanks for the reply, but it didn't work for me. I tried reinstalling stremio, cold restarting the TV, disconnecting the cord, changing DNS servers to google/cloudflare, completely resetting wifi settings, and then reconnecting. None of them worked.

**Comment by sergueidob** [comment]
> > > > Suddenly the same thing just happened to me. Cold booting my Samsung TV and reinstalling the app did not work. It was working yesterday. No updates or anything I'm aware of. Using Stremio + Torrentino + RD
> > > 
> > > 
> > > The same thing happened to me as well. I have the same setup with you.
> > > It works on pc & android phone on the same network but it doesn't work on my tizen tv. It stopped working this week. It gets stuck at loading screen when I want to load any stream, then shows "stream failed to load" message.
> > 
> > 
> > Try unpluggin the power cord and wait a few seconds, that did the trick for me. Using the remote control IS NOT THE SAME.
> > The TV must remain some time without power.
> 
> Thanks for the reply, but it didn't work for me. I tried reinstalling stremio, cold restarting the TV, disconnecting the cord, changing DNS servers to google/cloudflare, completely resetting wifi settings, and then reconnecting. None of them worked.

Hmm strange... Have you tried connecting your tv vía WiFi sharing your 3g-5g mobile Internet? Just yo make sure that your ISP is not blocking the app servers?

**Comment by GameDevUmut** [comment]
> Hmm strange... Have you tried connecting your tv vía WiFi sharing your 3g-5g mobile Internet? Just yo make sure that your ISP is not blocking the app servers?

Yes, I recently changed my ISP and it seems to be the source of the problem. I tried mobile wifi like you said and it worked on my TV. But the weird thing is that I can use stremio on pc and android phone with the same router and same ISP. Only Tizen version of Stremio has that problem.

**Comment by fpviviani** [comment]
> Suddenly the same thing just happened to me. Cold booting my Samsung TV and reinstalling the app did not work. It was working yesterday. No updates or anything I'm aware of. Using Stremio + Torrentino + RD

maybe it’s caused by some recent update? it was fine here but a few days ago it has gone off and nothing has helped. neither reinstalling, nor cold reboot.
“server” in settings is “offline” and reload doesn’t work.

theater 1.7.6
tyzen 8

**Comment by HElzomor** [comment]
Changing Torrentio worked for me to resolve the issue of "failed to load stream". I used Comet instead.

**Comment by michele95marini-hub** [comment]
> i flussi 1080p funzionano?

The same thing happens to me, but I only get 1080p.
what should I do?

**Comment by ibrahimennafaa** [comment]
Same issues. 
4k -> Stream failed to load
1080p -> works fine.

**Comment by DadyKay50** [comment]
Just force stop your tv stay pressing the bower botton till it's opend
again then try it

Le lun. 20 oct. 2025, 03:30, Ibrahim Ennafaa ***@***.***> a
écrit :

**Comment by Crunch123MM** [comment]
Is there a fix for this issue? I see the issues closed but I'm having the same problem and I've tried all the potential solutions but none of work.

**Comment by consecrated** [comment]
I had the exact same issue and at first I thought it was the TV or the app.

After a lot of testing, I found out the problem was actually my internet provider. My ISP was blocking both Stremio and IPTV.

I tried everything like changing DNS and other settings, but nothing worked. Using a VPN was not an option either, since the TV does not support it and I could not set it up on my network.

In the end, I switched to a different ISP and everything started working normally on the same TV.

So if anyone is facing this issue, it might not be the TV or the app, but the internet provider.
